### PR TITLE
Add support for organization debug certificate

### PR DESCRIPTION
**Pull request by oshtaier**
Adding one more helper for organization entity debug certificate and also put verification for response content type as we can receive different contents from server

### CODE REVIEW

File: nailgun/entities.py [hunk 1]
```diff
@@ -100,7 +100,9 @@ def _handle_response(response, server_config, synchronous=False):
     2. Poll the server for a foreman task to complete if an HTTP 202 (accepted)
        status code is returned and ``synchronous is True``.
     3. Immediately return if an HTTP "NO CONTENT" response is received.
-    4. Return the server's response, with all JSON decoded.
+    4. Determine what type of the content returned from server. Depending on
+    the type method should return server's response, with all JSON decoded or just
+    response content itself.
```
Comment (Ichimonji10) [inline nailgun/entities.py]: Try:

```
    4. Determine what type of the content returned from server. Depending on
       the type method should return server's response, with all JSON decoded or
       just response content itself.
```

File: nailgun/entities.py [hunk 2]
```diff
@@ -113,7 +115,10 @@ def _handle_response(response, server_config, synchronous=False):
         return ForemanTask(server_config, id=response.json()['id']).poll()
     if response.status_code == NO_CONTENT:
         return
-    return response.json()
+    if response.headers.get('content-type', '').lower() == 'application/json':
+        return response.json()
+    else:
+        return response.content
```
Comment (Ichimonji10) [inline nailgun/entities.py]: :+1:

**Comment by Ichimonji10** [comment]
Looks sane. Have you successfully used the method?

**Comment by oshtaier** [comment]
Yes, I got that certificate as a string and can operate with it in any way I like

File: nailgun/entities.py (hunk 1 above)
Comment (oshtaier) [inline nailgun/entities.py]: Thanks :)

**Comment by Ichimonji10** [comment]
ACK

Nice job, @oshtaier.

**Comment by abalakh** [comment]
ACK

**Pull request merged**